### ISSUE TITLE
Laplacian paths in complex networks: Information core emerges from entropic transitions

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#17]
## TL;DR

Paper propose a Laplacian renormalization group diffusion-based picture for complex networks, which is able to identify proper spatiotemporal scales in heterogeneous networks. Method validated through application to several real-world networks, demonstrating its ability to perform network reduction keeping crucial properties of the systems intact.

### Paper Link

https://www.nature.com/articles/s41567-022-01866-8
https://journals.aps.org/prresearch/pdf/10.1103/PhysRevResearch.4.033196

### Author/Institution

Pablo Villegas (Enrico Fermi’ Research Center), Tommaso Gili (IMT Scuola Alti Studi Lucca), Guido Caldarelli
 (University of Venice), Andrea Gabrielli (‘Enrico Fermi’ Research Center)

## Overview

A basic open question in statistical physics of networks is how to define a reduction scheme to detect all internal characteristic scales that significantly exceed the microscopic one. This is the hunting ground for a very powerful tool in modern theoretical physics, the renormalization group (RG. Here we propose a new diffusion-based RG scheme, taking advantage of the Laplacian operator, which detects appropriate spatiotemporal scales in heterogeneous networks.

## Methods

Network Science, dimensional reduction. gene expression data

## Cite

Villegas, P., Gili, T., Caldarelli, G. et al. Laplacian renormalization group for heterogeneous networks. Nat. Phys. 19, 445–450 (2023). https://doi.org/10.1038/s41567-022-01866-8

## Comments

I started writing R package for this in Rcpp called `infoCoRe`
